monitoring/rid_qualifier/host/mockuss.py

```diff
@@ -0,0 +1,12 @@
+import sys
+
+from monitoring.rid_qualifier.host import webapp
+
+
+def main(argv):
```

Comment:
Let's rename this file `debug_host.py` -- it's basically the debugging entrypoint into the `webapp` defined in this module (instead of using `gunicorn` like run_locally.sh does)

Reply:
Done.